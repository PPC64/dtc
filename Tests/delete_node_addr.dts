/dts-v1/;

/ {
    foo {
    };

    bar@42 {
    };
    bar {
    };
};

/ {
    /delete-node/ bar@42;
};

