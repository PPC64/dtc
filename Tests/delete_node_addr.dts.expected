/dts-v1/;

/  {

	foo {

	};
	bar {

	};
};
